example/node.js

```diff
@@ -1,10 +1,20 @@
-var fs = require('fs');
-var extract = require('..');
+const fs = require('fs');
+const extract = require('..');
+const { identifyProvisioningType } = require('../lib/provisioning');
+const { promisify } = require('util');
 
-var ipa = process.argv[2];
-var fd = fs.openSync(ipa, 'r');
+const openAsync = promisify(fs.open);
 
-extract(fd, function(err, info, raw){
-  if (err) throw err;
-  console.log(info);
+(async function main () {
+  const ipa = process.argv[2];
```

Comment:
Maybe worth checking for argv.length before going async and show a proper help message specifying the program expects an ipa as argument.

Apsrt from that, everything looks great to me and i think it’s time to merge :)

Reply:
it's an example ;-) the argument length is left to the reader ;-)

Reply:
then i'm fine for merging it